docs/ingressdns-with-externaldns.md

```diff
@@ -77,43 +77,37 @@ EOF
 Deploy test `FederatedDeployment`, `FederatedService` and `FederatedIngress` resources:
 ```bash
 cat <<EOF | kubectl create -f -
-apiVersion: v1
```

Comment:
I found that most of the YAML templates in `example/sample1` is in such format, how about delete following content for all of the examples? @marun @font 

```
apiVersion: v1
kind: List
items:
```